src/azure-cli/azure/cli/command_modules/acs/decorator.py

```diff
@@ -3301,22 +3301,64 @@ def get_cluster_autoscaler_profile(self) -> Union[Dict[str, str], None]:
     def get_uptime_sla(self) -> bool:
         """Obtain the value of uptime_sla.
 
+        This function will verify the parameter by default. If both uptime_sla and no_uptime_sla are specified, raise
+        a MutuallyExclusiveArgumentError.
+
         :return: bool
         """
         # read the original value passed by the command
         uptime_sla = self.raw_param.get("uptime_sla")
-        # try to read the property value corresponding to the parameter from the `mc` object
-        if (
-            self.mc and
-            self.mc.sku and
-            self.mc.sku.tier is not None
-        ):
-            uptime_sla = self.mc.sku.tier == "Paid"
+        # In create mode, try to read the property value corresponding to the parameter from the `mc` object
+        if self.decorator_mode == DecoratorMode.CREATE:
```

Comment:
do we have code path if the decorator_mode is other value?

Reply:
I mean. "blabla"

Reply:
Good catch, changed in commit [#d337899](https://github.com/Azure/azure-cli/pull/19699/commits/d33789913acfe271b036bd585ecf1d0049d72635).